README.md

```diff
@@ -105,6 +105,19 @@ The following environment variables must be set:
 - `TRACKER_WORKSPACE_ID`  
   This property configures the octane workspace id.
 
+- `TRACKER_API_URL`  
+    This property configures the issue tracker url
+
+#### Octane required configuration
+###### Note that the username and password must be generated for the shared_space and workspace
+
+- `TRACKER_SHARED_SPACE_ID`  
+  This property configures the octane shared space id.
+
+- `TRACKER_WORKSPACE_ID`  
+  This property configures the octane workspace id.
+
```

Comment:
This section is now in the ReadMe twice.

Reply:
Merge issue